docs/Manage-Space.rst

```diff
@@ -267,12 +267,16 @@ A description of each template is displayed below the selectors.
    -  Close: The user cannot request for joining, but only the space
       manager can invite him.
 
+
+.. note:: Template, Visbility and registration fields have a description of the choice selected to help you taking right decisions when creating a new space.
```

Comment:
```suggestion
.. note:: Template, Visibility and registration fields have a description of the choice selected to help you choosing the best template for your needs when creating a new space.
```

Reply:
Also, shouldn't fields be named as they appear on the screenshot to avoid confusion ?